# ===============================================
# 속도
# ===============================================

target_vel: 50.0
curve_vel: 40.0
curve_standard: 0.0045
lookahead_idx: 50
# ===============================================
# gain
# ===============================================
ld_gain: 0.5      # double ld = 5.0 + ld_gain * ego.vel;
k_gain: 1

# ===============================================
# pid
# ===============================================
Kp: 0.3
Ki: 0.02
Kd: 0

# ===============================================
# 경로 파라미터
# ===============================================

# 경로 파일 이름 (recorded path)
path_file_name: "/root/aim_ws/src/roscpp_morai/config/track_log_recorded.csv"

# 기준점 파일 이름
ref_file_name: "/root/aim_ws/src/roscpp_morai/config/ref.txt"